tests/test_user.py

```diff
@@ -127,17 +130,19 @@ def test_edit_as_superuser(
         cmd = f"{cmd} --make-superuser"
     if new_is_non_verified:
         cmd = f"{cmd} --remove-verified"
-    debug(cmd)
+
     res = invoke_as_superuser(cmd)
-    debug(res.data)
-    assert res.retcode == 0
-    assert res.data["email"] == NEW_EMAIL
-    assert res.data["cache_dir"] == NEW_CACHE_DIR
-    assert res.data["slurm_user"] == NEW_SLURM_USER
-    assert res.data["username"] == NEW_USERNAME
-    assert res.data["is_superuser"] == new_is_superuser
+
```

Comment:
In this test, let's make sure we also cover a successful call with `--make-verified`.
This can be done, e.g., with another `parametrize` line, for `new_is_verified`.